TeamSix/pacman.py

```diff
@@ -1,7 +1,9 @@
 import pygame
 
-WINDOW_HEIGHT = 900
+WINDOW_HEIGHT = 450
 WINDOW_WIDTH = 900
+STEP_SIZE = 30
```

Comment:
The size of a step is determined by the sizes of pacman and the window. If we use a constant for pacman's size parameters, (we should, instead of using magic numbers) we can calculate this value here instead of hardcoding.